docs/source/cpp/compute.rst

```diff
@@ -1445,6 +1445,8 @@ For timestamps inputs with non-empty timezone, localized timestamp components wi
 +--------------------+------------+-------------------+---------------+----------------------------+-------+
 | is_leap_year       | Unary      | Timestamp, Date   | Boolean       |                            |       |
 +--------------------+------------+-------------------+---------------+----------------------------+-------+
+| local_time         | Unary      | Timestamp         | Timestamp     |                            |       |
```

Comment:
Can we add a note?  I don't think it will be obvious what this function does.

Reply:
Good point. Added some text and moved under the `Timezone handling` chapter. I'll trigger a docs preview to make sure I've got the markup right.